.changeset/ten-schools-collect.md

```diff
@@ -0,0 +1,6 @@
+---
+"@rocket.chat/meteor": patch
+"@rocket.chat/i18n": patch
+---
+
+fixes missing information when user login having no 2FA set with a mandatory 2FA role
```

Comment:
Are you sure this PR is a fix? Was there anything broken before?
This PR looks to me like an improvement to the 2FA flow, thus, I would categorize it as a feature.